### ISSUE TITLE
Prince Keleseth's buff texture is missing

### ISSUE THREAD

**Issue by troggnostupidhs** [HearthSim/hs-bugs#796]
https://www.reddit.com/r/hearthstone/comments/6v8sif/til_prince_keleseths_buff_icon_is_missing/

> So I decided to play some of the new zoo with Prince Keleseth since I opened him a few days ago, and when I went to hover over one of the buffed minions, and instead of the icon being there it just says "Missing Ability Art 1."

![image](https://user-images.githubusercontent.com/22508641/29573313-a5a37b80-872c-11e7-8f08-71f1920fae09.png)

**Comment by troggnostupidhs** [comment]
https://www.reddit.com/r/hearthstone/comments/7191tb/til_prince_keleseths_buff_art_changes_to_whatever/

>I was playing NEW Pirate Warrior (haha it's not dead) when I noticed that my Keleseth-buffed Patches had Bloodsail Corsair's art on it.
It turned out that I had moused over my opponent's Bloodsail Corsair and that Keleseth's buff, which currently just has placeholder art, copied the art of that buff.
And then I found out that it would just copy the most recent looked-at buff on minions.
Here's some pictures of a few tests I ran.
There's some ambiguity with consistency. For example, if a minion has two different buffs with different arts, mousing from minion -> keleseth's buff will display the first buff image, but mousing from minion -> non-buffed minion -> keleseth's buff will display the second buff image. I haven't experimented with more yet.
I'm so confused why this happens #smallindiecompany
